README.md

```diff
@@ -2096,6 +2096,37 @@ ANSWERS.addComponent('SearchBar', {
 })
 ```
 
+The SDK also offers an `ANSWERS.registerTemplate` function. This is to register
+a template string to any given template name in our template bundle.
```

Comment:
Could "in our template bundle" be swapped out for phrasing that describes registering the template with the Handlebars compiler rather than the SDK's bundle itself?